packages/opc-header/README.md

```diff
@@ -0,0 +1,48 @@
+# This is opc-header 👋
+
+Web component based on lit-element
+
+https://docs.google.com/document/d/1PoxpPsret2njugkAG_YBmlS3ZNkDd0MTY-JZKDbesrQ/edit#heading=h.w85tasqfwpfc
+
+## Install
+
+```sh
+npm install
+```
+
+## Usage
+
+### Development server
+- Install packages
+
+```sh
+npm run install
```

Comment:
@anjneeksharma we might need, how to install and use the web component in this README